@xen-orchestra/backups/HealthCheckVmBackup.mjs

```diff
@@ -79,19 +80,12 @@ export class HealthCheckVmBackup {
                 vm.xenstore_data['vm-data/xo-backup-health-check'] === 'failure'),
             {
               timeout: remainingTimeout,
+              timeoutMessage: refOrUuid =>
+                `timeout reached while waiting for  ${refOrUuid} to report the startup script execution.`,
             }
           )
           const scriptOk = new Date()
           remainingTimeout -= scriptOk - guestToolsReady
-          if (remainingTimeout < 0) {
```

Comment:
The line above can also be removed no?
`remainingTimeout -= scriptOk - guestToolsReady`